subsys/bluetooth/host/hci_core.c

```diff
@@ -2400,7 +2400,7 @@ static void process_events(struct k_poll_event *ev, int count)
 #else
 #if defined(CONFIG_BT_ISO)
 /* command FIFO + MAX_ISO_CONN */
-#define EV_COUNT (1 + CONFIG_BT_ISO_MAX_CHAN)
+#define EV_COUNT (2 + CONFIG_BT_ISO_MAX_CHAN)
```

Comment:
If that is the case then the comment needs to be updated as well above.

Reply:
done